docs/configuring.md

```diff
@@ -703,6 +703,47 @@ questions with default answers.
     copier -fd 'user_name=Manuel Calavera' copy template destination
     ```
 
+!!! info
+
+    The `--data` option is mutually exclusive with [`--data-file`](#data_file) option.
```

Comment:
Juat a minor suggestion to retain the wording of the docs:

```suggestion
    The `-d, --data` flags are mutually exclusive with the [`--data-file`](#data_file) flag.
```

WDYT, @Ian2012?